modules/031-linstor/templates/piraeus-operator/rbac-for-us.yaml

```diff
@@ -167,6 +167,13 @@ rules:
       - watch
       - patch
       - update
+  # For rbac-proxy
+  - apiGroups: ["authentication.k8s.io"]
```

Comment:
You need to grant permissions by binding a common role instead of manually writing them. The role is `d8:rbac-proxy `.

Example:
https://github.com/deckhouse/deckhouse/blob/713e5722cb1afc4bcfc0715f99d826244611fea9/modules/300-prometheus/templates/prometheus/rbac-for-us.yaml#L94-L107